routes/cds-services.js

```diff
@@ -417,6 +418,26 @@ function getFHIRClient(req, res) {
   }
 }
 
+function addDiagnosticReportToBundle(customApiResponse, bundle, client) {
+  const orders = customApiResponse.Order ?? [];
+  console.log(`Orders count: ${orders.length}`);
+  orders.forEach(order => {
+    const searchURL = `DiagnosticReport?identifier=${order.OrderId}`;
```

Comment:
Confirmed in https://fhir.epic.com/Specifications?api=989 that Epic supports the `identifier` parameter in DiagnosticReport.Search (Results) (R4).